examples/mnist_gan.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import os
+import sys
+sys.path.append(os.getcwd())
+sys.path.append(os.path.join(os.getcwd(), 'test'))
+
+from tinygrad.tensor import Tensor, Function, register
+from tinygrad.utils import get_parameters
+import tinygrad.optim as optim
+
+GPU = os.getenv("GPU") is not None
+
+class LeakyReLU(Function):
```

Comment:
If
```
x.relu() + (-leaky_slope*x).relu()
```
works I'd merge this as a second class op

Reply:
let me check

Reply:
and u mean first class op right.

Reply:
(would need a test too)

Reply:
yea ofc

Reply:
worked